src/dotnet-archive/dotnet-archive.csproj

```diff
@@ -6,7 +6,7 @@
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
     <EnableApiCheck>false</EnableApiCheck>
-    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
The only issue with this is some issue with sqlclient cc @kichalla has more details.